CMakeLists.txt

```diff
@@ -79,11 +79,25 @@ if (ENABLE_ALPINE)
     add_subdirectory (alpine)
 endif ()
 
-option (ENABLE_UNIT_TESTS "Enable unit tests" OFF)
+option (ENABLE_UNIT_TESTS "Enable unit tests" ON)
```

Comment:
All the `ENABLE` options in IPPL are by default `OFF` as we follow the "opt in" way rather than  "opt out".
Although it may make sense for unit tests I am not sure if we want to change this behavior as it is a bit inconsistent with the other `ENABLE` options.

Reply:
https://github.com/IPPL-framework/ippl/pull/233/commits/7148da36893711cbca529b391806e5f0d574e238

Taken care of :)